src/app/locale/getLanguageService.ts

```diff
@@ -1,13 +1,30 @@
-import { createLanguageService, LanguageService } from '@bigcommerce/checkout-sdk';
-import { memoize } from '@bigcommerce/memoize';
+import { createLanguageService, LanguageConfig, LanguageService } from '@bigcommerce/checkout-sdk';
+import { merge } from 'lodash';
 
-import { DEFAULT_TRANSLATIONS } from './translations';
+import { FALLBACK_TRANSLATIONS } from './translations';
 
-function getLanguageService(): LanguageService {
-    return createLanguageService({
-        ...(window as any).language,
-        defaultTranslations: DEFAULT_TRANSLATIONS,
+let languageService: LanguageService | undefined;
+let fallbackLanguageService: LanguageService | undefined;
```

Comment:
Why use 2 copies? Can't we just reuse `languageService` for both?

```tsx
function getLanguageService(): {
  if (!languageService) {
    languageService = createLanguageService({
      defaultTranslations: FALLBACK_TRANSLATIONS,
    });
  }

  return languageService;
}
```

Reply:
No reason. I think I was experimenting and forgot to clean up. 😂